metrics/network/network-metrics-iperf3.sh

```diff
@@ -100,6 +106,48 @@ function iperf3_jitter() {
 	echo "Finish"
 }
 
+
+# Test/measure the bandwidth between the interconnection of a server/client.
+# A container takes the role of server and the host takes the role of client.
+function iperf3_host_container() {
+	local test_name="network host to container"
+
+	# Checks iperf3 tool installed in host
+	local cmds=("iperf3")
+	check_cmds "${cmds[@]}"
+
+	local server_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -s -V"
```

Comment:
General comment: all these tests are using the same mount args so it would make sense to define a variable for them to minimise the cost of maintaining the script. Also, I think there is scope to create a `start_iperf3` function that all the `iperf3` tests could use.

Reply:
Agree, a re-factor could help here - @MarioCarrilloA are you game to re-factor some of the common parts?